h/services/bulk_executor/_executor.py

```diff
@@ -31,8 +64,3 @@ def _assert_authority(self, field, value, embedded=False):
         raise InvalidDeclarationError(
             f"The {field} '{value}' does not match the expected authority"
         )
-
-    def _check_authority(self, data_type, body):
-        if data_type in (DataType.USER, DataType.GROUP):
-            self._assert_authority("authority", body.attributes["authority"])
-            self._assert_authority("query authority", body.query["authority"])
```

Comment:
This used to be a lot bigger, but it dwindled away to almost nothing, so I've just inlined it